### ISSUE TITLE
Suggestion - Show method used in user enumeration

### ISSUE THREAD

**Issue by ygirouard** [m4ll0k/WPSeku#6]
The script uses 3 methods to try to do user enumeration, but doesn't say which one it used to list the users when it prints the results. It would be nice to have the method used next to each user found.

Example:

|| ID: 0 - Name: multiadmin - Method: wp-json

**Comment by m4ll0k** [comment]
ok

**Comment by b4dnewz** [comment]
hi @ygirouard actually the scanner supports only [1 method](https://github.com/m4ll0k/WPSeku/blob/master/modules/discovery/users/wpusers.py#L32) which is querying the author page by ID.
I made a PR with the wp-json (REST API) users endpoint method which can enumerate up to one hundred results of "active" users who published content on the site.

But (personally) I think displaying the method along the result is unnecessary.
Once the PR is merged, if it is, I will try to implement it just to see how it goes and If can be really useful in case of many results or in case of testing own websites to know where the "leak" come from in order to fix it.
